### ISSUE TITLE
ObanWeb does not respect verbose setting

### ISSUE THREAD

**Issue by crbelaus** [sorentwo/oban#161]
### Environment

- Oban Version:
```
* oban 1.0.0 (Hex package) (mix)
  locked at 1.0.0 (oban) 029ab2cf
* oban_web 1.0.1 (Hex package) (mix)
  locked at 1.0.1 (hexpm:oban/oban_web) cf443306
```
- PostgreSQL Version
```
9.6.15
```
- Elixir & Erlang/OTP Versions (`elixir --version`)
```
Erlang/OTP 22 [erts-10.4.4] [source] [64-bit] [smp:8:8] [ds:8:8:10] [async-threads:1] [hipe]

Elixir 1.9.0 (compiled with Erlang/OTP 22)
```

### Current Behavior

I've just updated Oban from 0.7 to 1.0. When I start the server with `mix phx.server` or the console with `iex -S mix` I am seeing lots of queries like:

```
SELECT s0."node", s0."queue", coalesce(array_length(s0."running", 1), 0), s0."limit", s0."paused" FROM (SELECT o0."node" AS "node", o0."queue" AS "queue", o0."running" AS "running", o0."limit" AS "limit", o0."paused" AS "paused", rank() OVER "nq" AS "rank" FROM "oban_beats" AS o0 WHERE (o0."inserted_at" > $1) WINDOW "nq" AS (PARTITION BY o0."node", o0."queue" ORDER BY o0."inserted_at" DESC)) AS s0 WHERE (s0."rank" = 1) [~U[2020-01-29 14:38:10.712721Z]]
15:39:10.715 [debug] QUERY OK source="oban_jobs" db=0.3ms
SELECT o0."queue", o0."state", count(o0."id") FROM "oban_jobs" AS o0 GROUP BY o0."queue", o0."state" []
15:39:11.717 [debug] QUERY OK db=1.6ms
```

In my local configuration I have queues, logs and crontab disabled with:

```
config :hcmex, Oban, crontab: false, verbose: false, queues: false
```

### Expected Behavior

Having configured Oban with `verbose: false` I expect it not to log any queries.

**Comment by sorentwo** [comment]
Thanks for the report!

Those query logs are coming from ObanWeb, not Oban. It refreshes in the background whether there are any users viewing the dashboard.

I'll add a `verbose` option to the ObanWeb config to handle it.

**Comment by sorentwo** [comment]
This is fixed with ObanWeb v1.1.0 🎉
